app/worker/main.py

```diff
@@ -68,17 +70,20 @@ def create_archive_task(self, archive_json: str):
 
 @celery.task(name="create_sheet_task", bind=True)
 def create_sheet_task(self, sheet_json: str):
+    global AA_LOGGER_ID
     sheet = schemas.SubmitSheet.model_validate_json(sheet_json)
     queue_name = (create_sheet_task.request.delivery_info or {}).get(
         "routing_key", "unknown"
     )
     logger.info(f"[queue={queue_name}] SHEET START {sheet=}")
 
     args = get_orchestrator_args(
-        sheet.group_id, True, ["--gsheet_feeder.sheet_id", sheet.sheet_id]
+        sheet.group_id, True, ["--gsheet_feeder_db.sheet_id", sheet.sheet_id]
```

Comment:
Since this flag is used twice, a constant may make sense for later.

Reply:
fixed this, will auto approve and merge